pkgs/development/python-modules/baycomp/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, scipy
+, scikit-learn
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "baycomp";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-xDRywWvXzfSITdTHPdMH5KPacJf+Scg81eiNdRQpI7A=";
+  };
+
+  propagatedBuildInputs = [ numpy scipy scikit-learn matplotlib ];
+
+  pythonImportsCheck = [ "baycomp" ];
+
+  meta = {
+    description = "A library for Bayesian comparison of classifiers.";
```

Comment:
```suggestion
    description = "A library for Bayesian comparison of classifiers.";
```
```suggestion
    description = "A library for Bayesian comparison of classifiers";
```

Reply:
I really forgot about htis package because `git rebase --interactive` to that commit actually didn't show it's commit to edit xD